template/src/components/useChatUI.tsx

```diff
@@ -0,0 +1,54 @@
+/*
+********************************************
+ Copyright © 2021 Agora Lab, Inc., all rights reserved.
+ AppBuilder and all associated components, source code, APIs, services, and documentation 
+ (the “Materials”) are owned by Agora Lab, Inc. and its licensors. The Materials may not be 
+ accessed, used, modified, or distributed for any purpose without a license from Agora Lab, Inc.  
+ Use without a license or in violation of any license terms and conditions (including use for 
+ any purpose competitive to Agora Lab, Inc.’s business) is strictly prohibited. For more 
+ information visit https://appbuilder.agora.io. 
+*********************************************
+*/
+import {createHook} from 'fpe-implementation';
+import React,{createContext, SetStateAction} from 'react';
+
+export interface privateMsgLastSeenInterface {
+  userId: string | number,
+  lastSeenCount: number
+}
+export interface ChatUIDataInterface {
```

Comment:
Exporting this as it is, I'm not sure how helpful this is gonna be for the users, Add it as a pending task to be discussed later after the merge. 
TODO; Discuss

Reply:
Added in todo list